src/open-feature.ts

```diff
@@ -11,15 +11,15 @@ type OpenFeatureGlobal = {
 const _global = global as OpenFeatureGlobal;
 
 // TODO: make implement EvaluationLifeCycle, but statically...
-export class OpenFeature {
+export class OpenFeature implements EvaluationLifeCycle {
```

Comment:
Don't export this class at all, instead, just add something like this at the bottom of the file:

```
export const openFeature = OpenFeature.getInstance();
```

This way, we are EXPORTING an instance, and nobody should (nobody will be able) to call `OpenFeature.getInstance`.

Then, consuming code will just do:

`import { openFeature } from './open-feature.js'`, which will already be a singleton!